code/modules/mob/living/carbon/slime/slime.dm

```diff
@@ -204,6 +204,12 @@
 	updatehealth()
 	return
 
+/mob/living/carbon/slime/MouseDrop(var/atom/movable/A as mob|obj)
+	if(isliving(A) && A != usr)
+		var/mob/living/Food = A
+		if(Food.Adjacent(usr) && !stat && Food.stat != DEAD) //messy
```

Comment:
tested with this:
if(canUseTopic(Food, BE_CLOSE, NO_DEXTERY) && Food.stat != DEAD)
didn't seem to check for unconcious/dead, also could drag myself through windoors onto someone, that may have existed before though, i'll check that as well.